src/lib/molecules/Widget/index.scss

```
@@ -44,7 +44,7 @@
         grid-template-areas: 'image head' 'image text';
 
         &.t-default {
-            @include rtl(padding, 0 rem(20) 0 0, 0 0 0 rem(20));
+            @include rtl(padding, 0 rem(20), 0 rem(20));
```

We can change this line with:
```suggestion
         padding-inline: rem(20);
```